aws-sigv4/src/lib.rs

```diff
@@ -228,21 +241,121 @@ mod tests {
     }
 
     #[test]
-    fn test_build_authorization_header() -> Result<(), Error> {
+    fn test_set_xamz_sha_256() -> Result<(), Error> {
```

Comment:
Why the rename?

Reply:
I don't think this is a rename, but the addition of another test that confused the diff algorithm